### ISSUE TITLE
Master password isn't accepted

### ISSUE THREAD

**Issue by ykeremy** [bitwarden/clients#9953]
### Steps To Reproduce

```
➜ ~ BW_CLIENTID=user.xxx BW_CLIENTSECRET=xxx bw login --apikey
You are logged in!
...
➜ ~ BW_PASSWORD=xxx bw unlock --passwordenv BW_PASSWORD
Invalid master password
```

This same set of commands work on the previous CLI version. I'm sure that my master pwd is correct since it's the one I use on Bitwarden website.

### Expected Result

Vault should be unlocked.

### Actual Result

Invalid master password

### Operating System

macOS

### Operating System Version

ProductVersion:14.3 BuildVersion:23D2057

### Shell

Zsh

### Build Version

2024.6.1

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by djsmith85** [comment]
Duplicate of https://github.com/bitwarden/clients/issues/9919

**Comment by vpv-snbe** [comment]
Same in my environment.

bw.exe:
Version 2024.6.0

Bitwarden (self-hosted):
Version 2024.6.2

Master-Password via CLI is not been accepted

**Comment by imnamanmodi1** [comment]
+1

**Comment by snowy-jaguar** [comment]
Was this ever resilved? I seem to be running in to this issue

**Comment by pamperer562580892423** [comment]
@snowy-jaguar:

> Was this ever resilved? I seem to be running in to this issue

See here: https://github.com/bitwarden/clients/issues/9919

**Comment by snowy-jaguar** [comment]
> @snowy-jaguar:
> 
> > Was this ever resilved? I seem to be running in to this issue
> 
> See here: https://github.com/bitwarden/clients/issues/9919

Thanks, yeah after bashing my head against a wall for about an hour over this I decided to try logging out and back in and it worked. I think my issue was that my auth had silently timed out as I'd logged in months ago but the cli still thought I was logged in and I could retrieve some info but not add any. Even logging in with the api key during that timeout phase didn't work. But as soon as I logged out and back in again it started behaving as expected.
